apps/indexer/src/eventHandlers/voting.ts

```diff
@@ -64,8 +65,10 @@ export const voteCast = async (
 export const proposalCreated = async (
   context: Context,
   daoId: string,
```

Comment:
Runtime error: `proposalsOnchain.txHash` is defined as `notNull()` in the schema but the insert operation doesn't provide a value for `txHash`, which will cause database constraint violations.